core/src/main/java/com/linecorp/armeria/common/util/EventLoopThreadFactory.java

```diff
@@ -93,12 +118,22 @@ private EventLoopThreadFactory(ThreadFactory delegate) {
         this.delegate = delegate;
     }
 
+    EventLoopThreadFactory(ThreadFactory delegate,
+                           @Nullable Function<? super Runnable, ? extends Runnable> taskFunction) {
```

Comment:
We could remove `@Nullable` if we pass `Function.identity()` when task function is not specified in a builder.